testkube cli failed in Openshift environment

**Describe the bug**
Installing testkube successful but running failed in openshift env.
root@192-162-130-255:~# sudo apt-get install -y testkube
sudo: unable to resolve host 192-162-130-255
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following NEW packages will be installed:
  testkube
0 upgraded, 1 newly installed, 0 to remove and 149 not upgraded.
Need to get 0 B/15.5 MB of archives.
After this operation, 48.2 MB of additional disk space will be used.
Selecting previously unselected package testkube.
(Reading database ... 170084 files and directories currently installed.)
Preparing to unpack .../testkube_1.12.8_amd64.deb ...
Unpacking testkube (1.12.8) ...
Setting up testkube (1.12.8) ...

**To Reproduce**
Steps to reproduce the behavior:
1. Run 'testkube' or 'oc testkube'
2. Specify '...'
3. See error
root@192-162-130-255:~# testkube
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x5a9f54]

goroutine 1 [running]:
github.com/spf13/cobra.(*Command).Name(0x22c9480?)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1446 +0x14
github.com/kubeshop/testkube/cmd/kubectl-testkube/commands.glob..func1(0x31cdd40, {0x1ec0901?, 0x0?, 0x0?})
	/home/runner/work/testkube/testkube/cmd/kubectl-testkube/commands/root.go:68 +0x76
github.com/spf13/cobra.(*Command).execute(0x31cdd40, {0xc00003a240, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:920 +0x711
github.com/spf13/cobra.(*Command).ExecuteC(0x31cdd40)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/kubeshop/testkube/cmd/kubectl-testkube/commands.Execute()
	/home/runner/work/testkube/testkube/cmd/kubectl-testkube/commands/root.go:167 +0x37d
main.main()
	/home/runner/work/testkube/testkube/cmd/kubectl-testkube/main.go:24 +0x17
root@192-162-130-255:~# oc testkube
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x5a9f54]

goroutine 1 [running]:
github.com/spf13/cobra.(*Command).Name(0x22c9480?)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1446 +0x14
github.com/kubeshop/testkube/cmd/kubectl-testkube/commands.glob..func1(0x31cdd40, {0x1ec0901?, 0x0?, 0x0?})
	/home/runner/work/testkube/testkube/cmd/kubectl-testkube/commands/root.go:68 +0x76
github.com/spf13/cobra.(*Command).execute(0x31cdd40, {0xc00003a240, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:920 +0x711
github.com/spf13/cobra.(*Command).ExecuteC(0x31cdd40)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bc
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.7.0/command.go:992
github.com/kubeshop/testkube/cmd/kubectl-testkube/commands.Execute()
	/home/runner/work/testkube/testkube/cmd/kubectl-testkube/commands/root.go:167 +0x37d
main.main()
	/home/runner/work/testkube/testkube/cmd/kubectl-testkube/main.go:24 +0x17
**Expected behavior**
A clear and concise description of what you expected to happen.

**Version / Cluster**
- Which testkube version? 
root@192-162-130-255:~# testkube version

Context:  (1.12.8)   Namespace: testkube
----------------------------------------

████████ ███████ ███████ ████████ ██   ██ ██    ██ ██████  ███████
   ██    ██      ██         ██    ██  ██  ██    ██ ██   ██ ██
   ██    █████   ███████    ██    █████   ██    ██ ██████  █████
   ██    ██           ██    ██    ██  ██  ██    ██ ██   ██ ██
   ██    ███████ ███████    ██    ██   ██  ██████  ██████  ███████
                                           /tɛst kjub/ by Kubeshop


Client Version 1.12.8
Server Version  api/GET-testkube.ServerInfo returned error: api server response: '{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"services \"testkube-api-server\" not found","reason":"NotFound","details":{"name":"testkube-api-server","kind":"services"},"code":404}
'
error: services "testkube-api-server" not found
Commit 332b09fd03e17a56dafa0b5616a8866e8df6e3a7
Built by goreleaser
Build date 2023-06-20T12:51:41Z
⨯ getting config (error: api/GET-testkube.Config returned error: api server response: '{"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"services \"testkube-api-server\" not found","reason":"NotFound","details":{"name":"testkube-api-server","kind":"services"},"code":404}
'
error: services "testkube-api-server" not found)
- What Kubernetes cluster? (e.g. GKE, EKS, Openshift etc, local KinD, local Minikube)
- What Kubernetes version?

**Screenshots**
If applicable, add CLI commands/output to help explain your problem.
 
**Additional context**
Add any other context about the problem here.

--- a/cmd/kubectl-testkube/commands/root.go
+++ b/cmd/kubectl-testkube/commands/root.go
@@ -65,7 +65,7 @@ var RootCmd = &cobra.Command{
 		ui.SetVerbose(verbose)
 
 		// don't validate context before set and completion
-		if cmd.Name() == "context" || cmd.Parent().Name() == "completion" {
+		if cmd.Name() == "context" || (cmd.Parent() != nil && cmd.Parent().Name() == "completion") {
 			return
 		}
 

